src/Symfony/Component/Process/Pipes/WindowsPipes.php

```diff
@@ -71,7 +71,8 @@ public function __construct($input, bool $haveReadSupport)
                     }
                     $this->lockHandles[$pipe] = $h;
 
-                    if (!fclose(fopen($file, 'w')) || !$h = fopen($file, 'r')) {
+                    $fopen_result = fopen($file, 'w');
+                    if (false === $fopen_result || !fclose($fopen_result) || !$h = fopen($file, 'r')) {
```

Comment:
We don't use underscores in variable names. Also, let's keep the code compact here.

```suggestion
                    if (!($fileHandle = fopen($file, 'w')) || !fclose($fileHandle) || !$h = fopen($file, 'r')) {
```